doc/api/assert.markdown

```diff
@@ -2,8 +2,9 @@
 
     Stability: 2 - Stable
```

Comment:
FWIW, 3 is locked. Also, I thought there was talk about just getting rid of the stability index stuff altogether in the project. But I can't seem to find that right now. Maybe I imagined it...